cmd/state/internal/cmdtree/config.go

```diff
@@ -10,13 +10,13 @@ import (
 func newConfigCommand(prime *primer.Values) *captain.Command {
```

Comment:
We should mark it unstable, since technically this is a new command.

That means we're marking a command that used to be "stable" as "unstable", but given the original command was just printing help output I don't see a problem with that.

I'm pretty sure unstable doesn't propagate to child commands (ie. config set), but please make sure.

Reply:
It appears that we _do_ set the child commands to unstable as well. https://github.com/ActiveState/cli/blob/master/internal/captain/command.go#L410